src/server/server-html.jsx

```diff
@@ -40,14 +40,6 @@ export default function ServerHTML({ body, assets, locale, title, meta }) {
                                     content={m.content}
                                 />
                             );
-                        if (m.name && m.content)
```

Comment:
Any idea what's with this? Has it been returning dupe meta's this whole time?

Reply:
This was removed because we check it twice (why?)